prime-router/src/main/kotlin/history/azure/DeliveryFacade.kt

```diff
@@ -95,13 +95,27 @@ class DeliveryFacade(
      * @return Report details
      */
     fun findDetailedDeliveryHistory(
+        id: String,
         deliveryId: Long,
     ): DeliveryHistory? {
-        val deliveryHistory = dbDeliveryAccess.fetchAction(
-            deliveryId,
-            orgName = null,
-            DeliveryHistory::class.java
-        )
+        val reportFileId = try {
+            UUID.fromString(id)
+        } catch (ex: IllegalArgumentException) {
+            null
```

Comment:
log some kind of warning (but don't include the exception for now since it'll trigger pagerduty)

Reply:
I actually don't think we should log since its not unexpected that this could be an action id rather than a report UUID